api/filters/labels/example_test.go

```diff
@@ -29,6 +29,7 @@ metadata:
 		Filters: []kio.Filter{Filter{
 			Labels: map[string]string{
 				"foo": "bar",
+				"baz": "01234",
```

Comment:
Rather than changing existing test, could you add targeted new tests to `labels_test.go` in `TestLabels_Filter`? The examples here are meant to be the most basic cases and edge case tests should be targeted in their own specific tests.

Reply:
Yeah there's already a "number" test in labels_test.go so I'll just remove this.